components/definitions/definitions.hbs

```diff
@@ -0,0 +1,30 @@
+<div class="page">
+  <div class="page__content">
+    <div class="page__container container">
+      <div class="grid grid--reverse">
+        <div class="grid__col col-4@m"></div>
```

Comment:
This should have a `pull-1@m` added to place the panel over the 'gutter' column so that it matches the design.